src/Arcus.Template.WebApi/Controllers/HealthController.cs

```diff
@@ -27,14 +28,15 @@ public HealthController(HealthCheckService healthCheckService)
         }
 
         /// <summary>
-        ///     Get health of API application.
+        ///     Get Health
         /// </summary>
         /// <remarks>Provides an indication about the health of the API.</remarks>
         /// <response code="200">API is healthy</response>
         /// <response code="503">API is unhealthy or in degraded state</response>
         [HttpGet]
         [ProducesResponseType(typeof(HealthReport), StatusCodes.Status200OK)]
         [ProducesResponseType(typeof(HealthReport), StatusCodes.Status503ServiceUnavailable)]
+        [SwaggerOperation(OperationId = "Health_Get")]
```

Comment:
Let's move it to `[HttpGet(Name= "Health_Get")]` instead similar to in the link, if that's ok?